connect/src/main/java/com/stripe/android/connect/AccountOnboardingController.kt

```diff
@@ -13,12 +13,10 @@ class AccountOnboardingController internal constructor(
     embeddedComponentManager: EmbeddedComponentManager,
     title: String? = null,
     props: AccountOnboardingProps? = null,
-    cacheKey: String? = null,
-) : StripeComponentController<AccountOnboardingListener> by StripeComponentControllerImpl(
-    cls = AccountOnboardingDialogFragment::class.java,
+) : StripeComponentController<AccountOnboardingListener, AccountOnboardingProps>(
+    dfClass = AccountOnboardingDialogFragment::class.java,
```

Comment:
👀 
 1. As mentioned above, `cacheKey` is not necessary for Controllers since there can only be one DialogFragment per component type.
 2. `StripeComponentController` was converted to an abstract class with its implementation was migrated from `StripeComponentControllerImpl`